ceresdb/client.go

```diff
@@ -11,13 +11,12 @@ type CeresDBClient interface {
 	Query(context.Context, types.QueryRequest) (types.QueryResponse, error)
 	// Note: Rows currently writing to the same timeline will be overwritten; this restriction will be removed shortly
 	Write(context.Context, []*types.Row) (types.WriteResponse, error)
-	Close() error
```

Comment:
Why remove this?

Reply:
There is no `Close` method in java and rust sdk, so temporarily remove this method from golang sdk

Reply:
Close is useful to graceful close, so I think we don't need to follow other SDK style.

Reply:
Ok, I will add an issue and add the `Close` method as soon as possible